capstone/src/main/java/com/google/sps/data/RestaurantQueryHelper.java

```diff
@@ -34,7 +34,7 @@
    */
   public Restaurant makeRestaurantObject(Entity RestaurantEntity) throws NullPointerException {
     String name = (String) RestaurantEntity.getProperty("name");
-    int numberOfReviews = ((Long) RestaurantEntity.getProperty("numberOfReviews")).intValue();
+    int numberOfReviews = new Integer(RestaurantEntity.getProperty("numberOfReviews").toString());
```

Comment:
any specific reason we changed casts?

Reply:
Datastore stores all integer values as longs so need to cast from long --> int. For testing, I made an Entity object but did not pass it to Datastore, so its `numberOfReviews` field never got converted. So with the previous method, it would force cast it to a Long, but that could result in lossy conversion.

Both methods are doing the same thing, but second is more general.

Reply:
makes sense makes sense!